src/game_ui.c

```diff
@@ -1397,26 +1383,51 @@ void hud_main_time_trial(s32 arg0, Object *playerRacerObj, s32 updateRate) {
     if (get_contpak_error() > 0) {
         // has to be defined here to match
         char *SWMessage[3];
+
+#if REGION == REGION_JP
```

Comment:
Note: I can't seem to get make to run with a JP rom locally. Or rather I can't confirm nor deny whether this works as running `make REGION=jpn VERSION=v79` prints:
```
Linking: build/dkr.jpn.v79.elf
Objcopy: build/dkr.jpn.v79.elf -> build/dkr.jpn.v79.bin
CopyRom: build/dkr.jpn.v79.bin -> build/dkr.jpn.v79.z64
BootChip: CIC-NUS-6103
CRC 1: 0x53D440E7 Calculated: 0x53D440E7 (Good)
CRC 2: 0x7519B011 Calculated: 0x7519B011 (Good)
build/dkr.jpn.v79.z64: FAILED
sha1sum: WARNING: 1 computed checksum did NOT match
make: *** [Makefile:309: verify] Error 1
```

For both current master as well as this branch.